ci/prow/prow_setup.md

```diff
@@ -39,40 +47,55 @@
 
 1. Make sure that *Knative Robots* is an Admin of the repo.
 
-1. Update the tide section in the Prow config file and run `make update-config` (ask one of the owners of knative/test-infra).
+1. Update the tide section in the Prow config file and run `make update-config`
+   (ask one of the owners of knative/test-infra).
 
-1. Wait a few minutes, check that Prow is working by entering `/woof` as a comment in any PR in the new repo.
+1. Wait a few minutes, check that Prow is working by entering `/woof` as a
+   comment in any PR in the new repo.
 
 1. Set **tide** as a required status check for the master branch.
 
-### Setting up jobs for a new repo 
+   ![Branch Checks](branch_checks.png)
```

Comment:
this doesn't seem like belong to here

Reply:
It does. Why you think it doesn't?

Reply:
I saw the same pic showed up at line 101 also, is this intended?

Reply:
Yes. Same setting, different checks.